it/controllers/agentAppointment/ConfirmAgentAppointControllerISpec.scala

```diff
@@ -17,26 +17,38 @@ import java.util.UUID
 
 class ConfirmAgentAppointControllerISpec extends ISpecBase with HtmlComponentHelpers {
 
+  override lazy val extraConfig: Map[String, Any] = Map(
```

Comment:
Am I correct in thinking that the only new change here is the feature-switch = false. All others in this file is from formatting & stuff moving about?

Reply:
yes